scripts/jenkins/ardana/ansible/roles/ardana_tempest/tasks/main.yml

```diff
@@ -18,11 +18,19 @@
 - include_tasks: pre_defcore.yml
   when: tempest_run_filter == 'defcore'
 
+- include_tasks: reconfigure_neutron.yml
+  vars:
+    extra_vars: "-e router_distributed=False"
+  when: tempest_run_filter in ['vpnaas', 'fwaas']
+
 - include_tasks: import_octavia_image.yml
-  when: tempest_run_filter == 'lbaas'
+  when: tempest_run_filter in ['lbaas', 'heat']
```

Comment:
This change isn't covered by the commit message. Please make sure that it is not here by mistake.

Reply:
Sorry, seems like I missed that change, but yes it is intended.

heat tests also includes testing LBaaS resources, so we need the octavia image configured before running the heat tests.